### PR TITLE
Unify NuimosMenu and NuimoComponents screens - Issue 276

### LINKED ISSUE #276: Unify NuimosMenu and NuimoComponents screens on setup_app
Configuration menu has to be adapted in order to be in line with the design specified

### PR DESCRIPTION
**Pull request by commento**
Also, fix an issue related to multiple hue-bridges scenarios.

Closes #276

### CODE REVIEW

File: senic_hub/setup_app/components/AddComponent.js [hunk 1]
```diff
@@ -50,24 +50,23 @@ export default class AddComponent extends Screen {
         devices = response.devices
 
         var devIds = []
-        var bridgeIndex = null
-        //TODO: to be tested with multiple HUE BRIDGES
+        var usrIdx = null
```
Comment (rudolfs) [inline senic_hub/setup_app/components/AddComponent.js]: Let's use full variable names, it makes the code more readable.

File: senic_hub/setup_app/components/AddComponent.js [hunk 2]
```diff
@@ -50,24 +50,23 @@ export default class AddComponent extends Screen {
...
         //can be probably simplified
         for (var i = devices.length-1; i >= 0; i--){
           if (devices[i].type == 'philips_hue'){
-            if (!devices[i].id.includes('-light-')){
-              bridgeIndex = i
+            if (!devices[i].id.includes('-light-') && devices[i].id == usrIdx){
+              devices[i].id = devIds
+              devIds = []
```
Comment (rudolfs) [inline senic_hub/setup_app/components/AddComponent.js]: devIds => deviceIds

File: senic_hub/setup_app/components/AddComponent.js (hunk 1 above)
Comment (commento) [reply senic_hub/setup_app/components/AddComponent.js]: 6017f2f7d79bfbd66bd9ca8d31f2e328e3973ca9

File: senic_hub/setup_app/components/AddComponent.js (hunk 2 above)
Comment (commento) [reply senic_hub/setup_app/components/AddComponent.js]: 6017f2f7d79bfbd66bd9ca8d31f2e328e3973ca9

**Comment by commento** [comment]
Closes #276

**Pull request merged**